### ISSUE TITLE
implementing graphs to plugin

### ISSUE THREAD

**Issue by maxcomperatore** [ogrebenyuk/compilerexplorer#28]
what about we get the option for visualizing visually the asemmbly, would be awesome.

like: 
![cfg (5)](https://github.com/ogrebenyuk/compilerexplorer/assets/131000419/dfa92137-20fd-422d-8d24-bfd9853411f7)

**Comment by ogrebenyuk** [comment]
Yes, this looks great. Unfortunately, I have to time to work on something like that right now.

**Comment by maxcomperatore** [comment]
654 days have passed.
